element_calcium_imaging/imaging_no_curation.py

```diff
@@ -1009,18 +1007,16 @@ def make(self, key):
             if key["extraction_method"] == "suite2p_deconvolution":
                 suite2p_dataset = imaging_dataset
                 # ---- iterate through all s2p plane outputs ----
-                spikes = []
-                for s2p in suite2p_dataset.planes.values():
-                    mask_count = len(spikes)  # increment mask id from all "plane"
-                    for mask_idx, spks in enumerate(s2p.spks):
-                        spikes.append(
-                            {
-                                **key,
-                                "mask": mask_idx + mask_count,
-                                "fluo_channel": 0,
-                                "activity_trace": spks,
-                            }
-                        )
+                spikes = [
```

Comment:
you could now get rid of `spikes` and insert from the generator directly but it's only a minor improvement.